hack/keepalived-docker/manage-keepalived.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+export assigned_ip=172.22.0.2/24
+
+export interface=ironicendpoint
+while true;do 
+	/etc/init.d/keepalived start
+	ip a show $interface |grep "inet $assigned_ip"
+	if [ $? -gt 0 ]; then
+           echo "waiting for VIP to be assigned on interface $interface"
+	else
+           exit 0
+        fi
+	sleep 1
```

Comment:
why looping the start command ? Is it expected to fail for a while and then succeed ?

Reply:
Yes it takes time for keepallived to run. Restart doesn't work in this scenario. Might be some bug in keepalived